src/schema/v2/me/triggerCampaignMutation.ts

```diff
@@ -0,0 +1,37 @@
+import { GraphQLEnumType } from "graphql"
+import { GraphQLNonNull } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { ResolverContext } from "types/graphql"
+
+export const triggerCampaignMutation = mutationWithClientMutationId<
+  any,
+  any,
+  ResolverContext
+>({
+  name: "TriggerCampaign",
+  description: "Triggers a campaign send.",
+  inputFields: {
+    campaignID: {
+      type: new GraphQLNonNull(
+        new GraphQLEnumType({
+          name: "TriggerCampaignID",
+          values: {
+            // TODO: replace value w/ actual campaignID
+            ART_QUIZ: {
+              value: "art-quiz",
+            },
+          },
+        })
+      ),
+    },
+  },
+  outputFields: {},
+
+  mutateAndGetPayload: async (args, { triggerCampaignLoader }) => {
+    if (!triggerCampaignLoader) {
+      throw new Error("You need to be signed in to perform this action")
+    }
+
+    return triggerCampaignLoader({ campaign_id: args.campaignID })
+  },
+})
```

Comment:
@dzucconi - Matt Z has been flagging me on the ThingOrError pattern lately with new mutations. Thoughts?

Reply:
I dislike it a great deal, personally. If it was a good idea it should be part of the spec. But [here's an example](https://github.com/artsy/metaphysics/blob/main/src/schema/v2/me/mark_all_notifications_as_read_mutation.ts) @TMcMeans

Reply:
I dont have an opinion either way just wanna make sure I'm sticking to convention, thanks for the example

Reply:
love this.... but whats ChatGPT??

Reply:
(I feel weird suggesting this, but this is exactly what ChatGPT is good at, and I just did this for like 7 mutations and it worked great lol)

Reply:
@damassi https://openai.com/blog/chatgpt/??

Reply:
https://chat.openai.com

Reply:
Looks good!